sql/core/src/main/scala/org/apache/spark/sql/execution/joins/SortMergeJoinExec.scala

```diff
@@ -78,13 +78,21 @@ case class SortMergeJoinExec(
    * again, returns the required ordering for this child with extra "sameOrderExpressions" from
    * the child's outputOrdering.
    */
-  private def getKeyOrdering(keys: Seq[Expression], childOutputOrdering: Seq[SortOrder])
-    : Seq[SortOrder] = {
+  private def getKeyOrdering(
+      keys: Seq[Expression],
+      childOutputOrdering: Seq[SortOrder],
+      isStreamSide: Boolean = true) : Seq[SortOrder] = {
```

Comment:
1. `isStreamSide` -> `isKeepRemainingOrder`?
2. Please also update the comment.